src/app/archive/broadcasts/broadcasts_show.component.ts

```diff
@@ -127,20 +128,20 @@ export class BroadcastsShowComponent {
   private broadcastMoreObservable(): Observable<CrudList<BroadcastModel>> {
     return this.fetchMore
       .withLatestFrom(this.broadcastList, (_, list) => list)
-      .distinctUntilChanged(null, list => list.links.next)
-      .do(_ => this.fetchingMore = true)
-      .switchMap(list =>
+      .distinctUntilChanged(null, (list: CrudList<BroadcastModel>) =>
```

Comment:
Unable to find documentation about this `Observable#distinctUntilChanged()` method signature.

Official doc says it only has one param: http://reactivex.io/rxjs/class/es6/Observable.js~Observable.html#instance-method-distinctUntilChanged